chart/keep/values.yaml

```diff
@@ -102,6 +102,7 @@ database:
   enabled: true
   replicaCount: 1
   size: 5Gi
+  storageClasss: deafult
```

Comment:
@DmarshalTU 
is it default?
```suggestion
  storageClasss: default
```

Reply:
thank to point it out, it should be empty